examples/postInit/custom/vanilla.groovy

```diff
@@ -21,6 +21,10 @@ log.info(ore_iron >> (item_iron * 3)) // false
     println file.path
 }*/
 
+for (var stack in mods.theoneprobe.allItems) {
+    log.info stack
+}
```

Comment:
while `theoneprobe` is included in the development environment, this file is for vanilla exclusive stuff - what about this? using `ItemStack#getTranslationKey` to make what is being printed more readable.
```suggestion
for (var stack in mods.minecraft.allItems.getItemStacks()[5..12]) {
    log.info stack.getTranslationKey()
}
```
side note: a bit surprised that `ItemsIngredient` doesnt implement `List` (it does implement `Iterable`)